packages/dendron-cli/.vscode/tasks.json

```diff
@@ -64,6 +64,36 @@
       "type": "shell",
       "command": "node --inspect ${workspaceFolder}/lib/bin/dendron-cli publishPod --wsRoot ${workspaceFolder}/../../test-workspace --podId dendron.markdown --vault vault --query root --podSource builtin",
       "problemMatcher": []
-    }
+    },
+    {
+      "label": "markdown-export-v2",
```

Comment:
just a reminder to add this to [[Test|dendron://dendron.docs/pkg.pods-core.qa.test]] :)

Reply:
Had a PR in review for this under [[dendron://dendron.docs/dev.process.qa.cli]], Should I update to this ⬆️ ?
https://github.com/dendronhq/dendron-docs/pull/17

Reply:
a link mentioning it would be great :)